internal/jpcommon/map.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package jpcommon
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+
+	"github.com/jaegertracing/jaeger/proto-gen/storage/v2"
+)
+
+func PcommonMapToPlainMap(attributes pcommon.Map) map[string]string {
+	mapAttributes := make(map[string]string)
+	attributes.Range(func(k string, v pcommon.Value) bool {
```

Comment:
nit: I recommend using the new iterator methods, I think it will be attributes.All()

Reply:
unless they still haven't merged those, I thought they did in the last release